test/e2e/volume_provisioning.go

```diff
@@ -90,11 +129,80 @@ func testDynamicProvisioning(client clientset.Interface, claim *v1.PersistentVol
 
 	// Wait for the PV to get deleted. Technically, the first few delete
 	// attempts may fail, as the volume is still attached to a node because
-	// kubelet is slowly cleaning up a pod, however it should succeed in a
-	// couple of minutes. Wait 20 minutes to recover from random cloud hiccups.
+	// kubelet is slowly cleaning up the previous pod, however it should succeed
+	// in a couple of minutes. Wait 20 minutes to recover from random cloud
+	// hiccups.
 	framework.ExpectNoError(framework.WaitForPersistentVolumeDeleted(client, pv.Name, 5*time.Second, 20*time.Minute))
 }
 
+// checkAWSEBS checks properties of an AWS EBS. Test framework does not
+// instantiate full AWS provider, therefore we need use ec2 API directly.
+func checkAWSEBS(volume *v1.PersistentVolume, volumeType string, encrypted bool) error {
```

Comment:
No, the test framework does not instantiate AWS cloud provider. See e.g. https://github.com/kubernetes/kubernetes/blob/master/test/e2e/pd.go#L574

Yes, it *should* use cloud provider, but that's for a separate PR and to be honest, I don't know how to start one. @justinsb, how hard would it be?